app/eventyay/config/settings.py

```diff
@@ -828,8 +842,11 @@ def instance_name(request):
 
 STATIC_URL = config.get('urls', 'static', fallback=BASE_PATH + '/static/')
 
-STATICFILES_DIRS = [os.path.join(BASE_DIR, 'static')]
-
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, 'webapp', 'dist'),
```

Comment:
I think `python manage.py collectstatic` will find the files in `webapp/dist` if added to `STATICFILES_DIR` as in
```
STATICFILES_DIRS.append(FRONTEND_DIR / 'schedule-editor' / 'dist')
```